api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/PaymentCycleScheduler.java

```diff
@@ -17,6 +18,7 @@
  * This scheduler simply invokes the CreateNewPaymentCycleJob for each such claim.
  */
 @Slf4j
+@Component
```

Comment:
The only reason the MessageProcessor didn't need `@Component` is because it is created from config by an `@Bean` annotated method.